src/cli/webpack.config.js

```diff
@@ -28,17 +28,10 @@ const config = {
 		publicPath: '/',
 	},
 	devServer: {
-		contentBase: path.resolve( process.cwd(), 'dist' ),
+		static: path.join( __dirname, 'dist' ),
 		historyApiFallback: true,
 	},
 	resolve: {
-		modules: [
-			path.relative(
-				process.cwd(),
-				path.resolve( __dirname, '../../node_modules' )
-			),
-			path.resolve( process.cwd(), 'node_modules' ),
-		],
```

Comment:
Thanks for the PR, this is the only thing that I'm not sure about, the idea was that I wanted to ensure the modules from blockbook (WP modules) have more priority than the ones that come from the repo of the theme/plugin. Do you know if this is still the case and how is this achieved?

Reply:
The issue I was hitting here is that with this included, webpack was not resolving nested node_modules. For example, `react-autosize-textarea`, which was installed in `blockbook/node_modules/@wordpress/block-editor/node_modules` wasn't being found.

Reply:
My read of https://nodejs.org/docs/latest-v14.x/api/modules.html#modules_loading_from_node_modules_folders is that Node starts with the parent of the current location and works outward only if the module isn't found, so I think blockbook modules would have priority over ones in the theme or plugin.